ios/FluentUI.Demo/FluentUI.Demo/Demos/SegmentedControlDemoController.swift

```diff
@@ -38,20 +38,54 @@ class SegmentedControlDemoController: DemoController {
         container.addArrangedSubview(disabledTabsSegmentedControl)
         container.addArrangedSubview(UIView())
 
-        addTitle(text: "Switch")
+        addTitle(text: "Primary Pill")
```

Comment:
for line 25 and line 33 can we append "depcreated" in title for the example of tabs